app/addons/fauxton/base.js

```diff
@@ -104,7 +104,7 @@ function (app, FauxtonAPI, Components, NavbarReactComponents, NavigationActions,
 
   Fauxton.Notification = FauxtonAPI.View.extend({
     animationTimer: 5000,
-
+    id: 'global-notification-id',
```

Comment:
Meaning, there are several nested elements inside the global notifications banner. All without IDs. This add one ID.